mail_cleanup/__openerp__.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Matthieu Dietrich
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Mail cleanup',
+    'version': '0.1',
+    'category': 'Tools',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'summary': 'Clean up mails regularly',
+    'description': """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Mail cleanup
+===========
+
+This module allows to mark e-mails older than x days as read
+and optionally move them on IMAP servers, just before fetching them.
+
+Since the main "mail" module does not mark unroutable e-mails as read,
+this means that if junk mail arrives in the catch-all address without
+any default route, fetching newer e-mails will happen after re-parsing
+those unroutable e-mails.
```

Comment:
It was due to a previous comment asking for the README, before realizing this is for 7.0.
I'll remove the README.